src/marlin/marlin_pst13_pc/combinations.rs

```diff
@@ -0,0 +1,138 @@
+//! Compute all combinations of values in a given list
+//! Credit: https://github.com/meltinglava/uniquecombinations/
```

Comment:
I checked. This repo is under Apache 2 and MIT license, so it fits well into our license. Any additional reference should we put?